src/main/scala/com/kubukoz/next/Main.scala

```diff
@@ -87,7 +88,7 @@ object Main extends CommandIOApp(name = "spotify-next", header = "spotify-next:
     def reportError(e: Throwable): IO[Unit] =
       Console[IO].errorln("Command failed with exception: ") *> IO(e.printStackTrace())
 
-    fs2.Stream.exec(IO.println("Loading REPL...")) ++
+    fs2.Stream.exec(IO.println(s"Loading REPL...version: ${BuildInfo.version.takeWhile(_ != '+')}")) ++
```

Comment:
IMO we can print the full version. There isn't going to be a `+` in the tagged releases :)

Reply:
Also, about the format, I would print this in the welcome message:

```
Loading REPL...
Welcome to the spotify-next ${BuildInfo.version} REPL! Type in a command to begin
```

This is a couple lines below.